site/src/styles/page-chips.css

```
@@ -0,0 +1,87 @@
+  line-height: 1.5;
+  white-space: nowrap;
+  text-decoration: none;
+  transition:
+    background-color 0.12s ease,
+    border-color 0.12s ease;
+}
+
+.page-chips .chip-standard {
+  --chip-line: var(--ph-chip-standard-line);
+  --chip-fill: var(--ph-chip-standard-fill);
+  color: var(--ph-chip-standard-ink);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)